### PR TITLE
Add nomenclature changes notification banner

### PR DESCRIPTION
**Pull request by Levia**
Add nomenclature changes notification banner to public interface.
Wording to be amended.

### CODE REVIEW

**Comment by agnessa** [comment]
hi! this needs to go only on pages for species affected by the nomenclature change, and only for a given period of time, I think we discussed 6 months (that needs to be configurable). I would first have a go at establishing this automatically, e.g. through a check if a taxon or its ancestor was an output of a nomenclature change within the last 6 months?

**Comment by Levia** [comment]
Ah sorry, I've got this mistaken. So by configurable you mean that we (developers) can amend the range whenever requested? Or maybe that it's a configurable field by any admin in the system?

**Comment by Levia** [comment]
I've done some amendments following your comment. Thank you!

File: app/serializers/species/show_taxon_concept_serializer.rb [hunk 1]
```diff
@@ -119,4 +119,12 @@ def cache_key
     Rails.logger.debug "CACHE KEY: #{key.inspect}"
     key
   end
+
+  def nomenclature_notification
+    outputs = NomenclatureChange::Output.where(taxon_concept_id: object.id)
+    return false unless outputs
+
+    outputs.where("created_at < #{6.months.ago}")
```
Comment (agnessa) [inline app/serializers/species/show_taxon_concept_serializer.rb]: This part of the query is not assigned to `outputs`, so it won't have effect on the returned result. Perhaps could be simplified by just keeping the query in one chain (not sure if there's benefit to splitting it?) and returning `outputs.present?`

Also, please add tests for this logic and consider whether we might need to check for both `taxon_concept_id` and `new_taxon_concept_id`. For example, if we lump A and B into C, then when looking at C's page we want to see the banner - and C's `id` is going to be stored in `new_taxon_concept_id` I believe.

**Comment by Levia** [comment]
I've made some amendments and added some tests

**Pull request merged**